python/nano/src/bigdl/nano/pytorch/torch_nano.py

```diff
@@ -208,11 +213,27 @@ def __init__(self, num_processes: Optional[int] = None,
             else:
                 strategy = None     # type: ignore
         elif distributed_backend in backends_class_map:
+            if process_group_backend is not None:
```

Comment:
nitpick: maybe we can move these lines to a `_check_ccl` function instead of replicating in `torch_nano.py` and `trainer.py`